eclass/tree-sitter-grammar.eclass

```diff
@@ -206,6 +208,19 @@ tree-sitter-grammar_src_install() {
 
 		dolib.so "${WORKDIR}/${soname}"
 		dosym "${soname}" /usr/$(get_libdir)/lib${PN}$(get_libname)
+		# Install symlinks to grammars so that they can be found by NeoVim.
+		# /usr/$(get_libdir)/tree-sitter gets added to the NeoVim runtimepath.
+		# See neovim/runtime/doc/treesitter.txt for info.
+		keepdir /usr/$(get_libdir)/tree-sitter
+		dosym ../"${soname}" \
+			/usr/$(get_libdir)/tree-sitter/parser/${PN##tree-sitter-}$(get_libname)
```

Comment:
Looks good. But just curious, and what about emacs ant others?
Where do they search for parsers now? :thinking:

Reply:
Emacs looks on the library path. I think it's fine to only mention Neovim here as this seems to be something they've come up with.